### DISCUSSION TITLE
What is the best way to access local data like translation files

### DISCUSSION THREAD

**Discussion by muuvmuuv** [ionic-team/capacitor#4724] [Q&A]
We have an Angular Ionic app and use ngneat/transloco to serve translations. At the moment we are using httpClient to receive translations which are in public folder in the app but we want to switch away completely from httpClient. How is anyone doing local file getting in Ionic or Capacitor especially? We tried with FileSystem but this seems to only have access to user data not app data?

**Comment by jcesarmobile** [comment]
if it's in the public folder you can get them with XHR/fetch

https://developers.google.com/web/updates/2015/03/introduction-to-fetch

**Reply by muuvmuuv** [reply]
Good point, to use WebAPI instead of Angular httpClient Wrapper. But does that mean I have no access through native API's into public folder?
